Sofee.Core.Api/Models/Languages/Exceptions/LanguageValidationException.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using Xeptions;
+
+namespace Sofee.Core.Api.Models.Languages.Exceptions
+{
+    public class LanguageValidationException : Xeption
+    {
+        public LanguageValidationException(Xeption innerException)
+            : base(message: "Language validation error occured. Please, try again.", 
+                       innerException: innerException)
```

Comment:
Push this back to the `message` level since they are both input variable of the same method here.